tensorboard/webapp/runs/store/utils.ts

```diff
@@ -81,3 +81,21 @@ export function groupRuns(
   }
   return runGroup;
 }
+
+/**
+ * Util function for composing `GroupBy` key and regex string.
+ */
+export function createGroupBy(
+  groupByKey: GroupByKey,
+  regexString: string | null
```

Comment:
Can `regexString` be `null` in Reducer? If not, is it important that it is `null`able in this utility?

Reply:
removed `null` type.
Yet I think `regexString` could be an optional input. add `?` and replaced with empty string if it does not exist.